src/components/Attachments/AttachmentCarousel/Pager/AttachmentCarouselPagerContext.ts

```diff
@@ -3,7 +3,15 @@ import {createContext} from 'react';
 import type PagerView from 'react-native-pager-view';
 import type {SharedValue} from 'react-native-reanimated';
 
+type AttachmentCarouselItemMeta = {
+    source: string;
+    index: number;
+    isActive: boolean;
```

Comment:
Can you add a comment about what all these are used for? Also, do we need the Meta part here as well?